native/cocos/platform/openharmony/napi/NapiHelper.cpp

```diff
@@ -41,9 +42,15 @@ enum ContextType {
     NATIVE_RENDER_API,
     WORKER_INIT,
     ENGINE_UTILS,
+    EDITBOX_UTILS,
     UV_ASYNC_SEND
 };
 
+extern napi_value napiSetShowEditBoxFunction(napi_env env, napi_callback_info info);
+extern napi_value napiSetHideEditBoxFunction(napi_env env, napi_callback_info info);
+extern napi_value napiOnTextChange(napi_env env, napi_callback_info info);
+extern napi_value napiOnComplete(napi_env env, napi_callback_info info);
```

Comment:
Why use extern?

Reply:
Because ui/edit-box is not suitable for adding interfaces. And editbox is a module, so I want to put it in editbox-openharmony.cpp

Reply:
I think it is better to:

- add `editbox-openharmony.h` to declare editbox related functions
- implement NAPI functions here and invoke editbox related functions

I also have another question: should wrap theses codes into `CC_USE_EDITBOX `?

Reply:
OK